README.md

```diff
@@ -1,103 +1,14 @@
-# 100-temporal-mistakes
+# 100 Temporal Mistakes
 
 A collection of common mistakes made when using [Temporal](https://temporal.io) and how to avoid them. This is a work in progress!
 
-## Workflow Limits
-- Overflowing workflow history length
-- Overflowing workflow history bytes
-- Overflowing maximum individual payload size
-- Shard contention due to concurrent updates
-	- Concurrent activities
-	- Concurrent heartbeats
-	- Concurrent child workflows
-	- Frequent signals, updates, queries
-- Exceeding 10s task timeout
+## 1. Workflow Limits
 
-## Workflow Replay
-- Not using workflow versioning/patching
-- Incorrect workflow patching (wrong version numbers, removing branch before it's safe)
-- Thinking that replay means re-running activities
-- Performing network calls in workflow code
-- Using system time instead of workflow time
-- Reading environment variables in workflow code
-- Modifying shared state in workflow code
-- Performing expensive computation in workflow code
-- Passing too much information from activities to workflow code
-- Lossy payload serialization/deserialization
-- Breaking changes to payloads
-- Not using the return value in a side effect
-- Not using the Temporal SDK for logging, metrics, tracing
-- Not using workflow replay for debugging
-- Not using static analysis / sandboxed SDK
-- Modifying workflow history in interceptors or shared libraries
+- [WL0001 • Overflowing Workflow History Size](wl0001-overflowing-workflow-history-size.md)
```

Comment:
I'd like to skip adding numerical identifiers for now so that we can reorder entries or even re-categorize them as needed.

```suggestion
- [Overflowing Workflow History Size](overflowing-workflow-history-size.md)
```